ingest-app/src/cmr/ingest/api/routes.clj

```diff
@@ -74,6 +74,11 @@
            (acl/verify-ingest-management-permission ctx :update)
            (jobs/trigger-partial-refresh-collection-granule-aggregation-cache
             ctx)
+           {:status 200})
+     (POST "/trigger-granule-task-cleanup-job"
+           {ctx :request-context}
+           (acl/verify-ingest-management-permission ctx :update)
+           (jobs/bulk-granule-update-task-table-cleanup ctx)
```

Comment:
I think you understood me. I asked you to just call the cleanup function in the background job. That does not mean we should also do this for the endpoint. As we don't know how long this delete is going to take, I think it makes sense to make this asynchronous via the event.